UnitTests/TextFormatterTests.cs

```diff
@@ -3334,6 +3334,35 @@ public void Draw_Vertical_Wide_TextAlignments ()
 			Assert.Equal (new Rect (0, 0, 13, height + 2), pos);
 		}
 
+		[Fact, AutoInitShutdown]
+		public void Draw_Fill_Remaining ()
+		{
+			var view = new View ("A very long text behind the TextFormatter.");
+
+			var tf = new TextFormatter ();
+			tf.Text = "Hello";
+			tf.Size = new Size (10, 1);
+
+			Application.Top.Add (view);
+			Application.Begin (Application.Top);
+
+			GraphViewTests.AssertDriverContentsWithFrameAre (@"
+A very long text behind the TextFormatter.
+", output);
+
+			tf.Draw (new Rect (0, 0, 10, 1), view.GetNormalColor (), view.ColorScheme.HotNormal, default, false);
+			Application.Top.Redraw (Application.Top.Bounds);
+			GraphViewTests.AssertDriverContentsWithFrameAre (@"
+Helloy long text behind the TextFormatter.
+", output);
+
+			tf.Draw (new Rect (0, 0, 10, 1), view.GetNormalColor (), view.ColorScheme.HotNormal);
+			Application.Top.Redraw (Application.Top.Bounds);
+			GraphViewTests.AssertDriverContentsWithFrameAre (@"
+Hello     g text behind the TextFormatter.
```

Comment:
I don't understand why anyone would want "A very long text behind the TextFormatter." to be formatted as "Hello     g text behind the TextFormatter."

Reply:
I know it's only to explain that if we want the view width been respected the remaining will be filled by spaces. Imagine we want draw listview lines with it and leverage the alignment features.

Reply:
Well that was what comes up on my mind at the time. What do you suggest to change this test?

Reply:
I rewritten the unit test and I hope you can understand the main reason of this PR. With the `tf2` I cleared and written at once.